FirstProject/evaluation.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" href="img/">
+        <!-- Latest compiled and minified CSS -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+
+        <!-- jQuery library -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+        <!-- Latest compiled JavaScript -->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    </head>
+
+    <body>
+        <div>
+            <div class="row">
+                <nav class="col-md-12" style="border-bottom: 2px solid darkgrey; padding-top: 12px; padding-bottom: 14px;">
```

Comment:
In programming we have a concept called "borrowed time" which refers to doing something fast and cutting corners with the mindset of coming back to it at a later point. I would advise against developing this behavior because in many instances you will not have time to come back to it at a later point... or someone else will come back to it and it will have a lot more work to do. It takes just as much time to do it the right way from the first go and that is something you guys should concentrate towards.